kube/destroy-info-schema.yaml

```diff
@@ -12,7 +12,7 @@ spec:
     spec:
       containers:
       - name: toolbox
-        image: quay.io/ukhomeofficedigital/hocs-toolbox:branch-main # TODO: make this :latest
+        image: quay.io/ukhomeofficedigital/hocs-toolbox:latest
```

Comment:
I think you can just drop :latest here, and it'll do that by default

Reply:
Done - also set the imagePullPolicy to Always to ensure that we get the new latest build